### ISSUE TITLE
Window not remembering window size and position

### ISSUE THREAD

**Issue by MikeCheel** [Murmele/Gittyup#350]
Every time I open the app the window is way too big and doesn't remember the new position and size after closing. I have to use ALT + SPACE and the Move to get to the window handles.

**Comment by exactly-one-kas** [comment]
Which operating system are you using?

**Comment by Alliegaytor** [comment]
I have the same issue. It always opens at the same size and doesn't remember what it used to be sized as.
Also happens on my laptop running KDE & Arch.

version (commit): 1.2.0 (b663589)

screenfetch:
<pre><span style="color:#AA5500"><b>                  +</b></span><span style="color:#AA00AA">I</span><span style="color:#00AAAA">+</span>		<span style="color:#00AAAA">        </span><span style="color:#00AAAA"><b> allison</b></span><b>@</b><span style="color:#00AAAA"><b>allison-ms7c37</b></span>
<span style="color:#AA5500"><b>                 +</b></span><span style="color:#AA00AA">777</span><span style="color:#00AAAA">+                  </span><span style="color:#00AAAA"><b> OS:</b></span> EndeavourOS rolling rolling
	<span style="color:#AA5500"><b>        +</b></span><span style="color:#AA00AA">77777</span><span style="color:#00AAAA">++</span>		<span style="color:#00AAAA"><b> Kernel:</b></span> x86_64 Linux 6.1.1-1-clear
	<span style="color:#AA5500"><b>       +</b></span><span style="color:#AA00AA">7777777</span><span style="color:#00AAAA">++</span>		<span style="color:#00AAAA"><b> Uptime:</b></span> 1h 56m
	<span style="color:#AA5500"><b>      +</b></span><span style="color:#AA00AA">7777777777</span><span style="color:#00AAAA">++</span>		<span style="color:#00AAAA"><b> Packages:</b></span> 1619
	<span style="color:#AA5500"><b>    ++</b></span><span style="color:#AA00AA">7777777777777</span><span style="color:#00AAAA">++</span>		<span style="color:#00AAAA"><b> Shell:</b></span> fish 3.5.1
	<span style="color:#AA5500"><b>   ++</b></span><span style="color:#AA00AA">777777777777777</span><span style="color:#00AAAA">+++       </span>	<span style="color:#00AAAA"><b> Resolution:</b></span> 2560x1440
	<span style="color:#AA5500"><b> ++</b></span><span style="color:#AA00AA">77777777777777777</span><span style="color:#00AAAA">++++        </span><span style="color:#00AAAA"><b> DE:</b></span> Xfce4
	<span style="color:#AA5500"><b>++</b></span><span style="color:#AA00AA">7777777777777777777</span><span style="color:#00AAAA">++++       </span><span style="color:#00AAAA"><b> WM:</b></span> Xfwm4
<span style="color:#AA5500"><b>      +++</b></span><span style="color:#AA00AA">777777777777777777777</span><span style="color:#00AAAA">++++</span>	<span style="color:#00AAAA"><b> WM Theme:</b></span> Aly
<span style="color:#AA5500"><b>    ++++</b></span><span style="color:#AA00AA">7777777777777777777777</span><span style="color:#00AAAA">+++++  </span>	<span style="color:#00AAAA"><b> GTK Theme:</b></span> Aly [GTK2]
<span style="color:#AA5500"><b>   ++++</b></span><span style="color:#AA00AA">77777777777777777777777</span><span style="color:#00AAAA">+++++  </span>	<span style="color:#00AAAA"><b> Icon Theme:</b></span> Qogir
<span style="color:#AA5500"><b>  +++++</b></span><span style="color:#AA00AA">777777777777777777777777</span><span style="color:#00AAAA">+++++</span>	<span style="color:#00AAAA"><b> Font:</b></span> Noto Sans 10
<span style="color:#00AAAA">       +++++++</span><span style="color:#AA00AA">7777777777777777</span><span style="color:#00AAAA">++++++</span>	<span style="color:#00AAAA"><b> Disk:</b></span> 1.5T / 2.4T (66%)
<span style="color:#00AAAA">      +++++++++++++++++++++++++++++     </span><span style="color:#00AAAA"><b> CPU:</b></span> AMD Ryzen 9 5900X 12-Core @ 24x 3.7GHz
<span style="color:#00AAAA">     +++++++++++++++++++++++++++        </span><span style="color:#00AAAA"><b> GPU:</b></span> AMD Radeon RX 5700 XT (navi10, LLVM 14.0.6, DRM 3.49, 6.1.1-1-clear)
                                      	<span style="color:#00AAAA"><b> RAM:</b></span> 7435MiB / 32020MiB
</pre>

**Comment by goktugerol-dev** [comment]
I agree with this bug. I use Gittyup on my Mac in the office, and it works perfectly fine. However, I also installed Gittyup on my Windows 11 laptop, and the resolution makes it unusable. It doesn't fit the screen properly; the upper bar is out of the screen, and the texts are extremely big. As a result, I uninstalled it from my laptop until it gets fixed. I'll keep using it on macOS.

**Comment by doctorjei** [comment]
Came here to mention the Windows 11 window sizing issue. The window starts up too large, and it doesn't save window size, so every time I open the app, I have to use keyboard shortcuts to move the window, then resize it. It's painful.

**Comment by goktugerol-dev** [comment]
Not only the size but also doesn't save the configuration... Everytime I open I have to setup my repos and projects again and again. On Mac there is no problem at all, I think it only happens on Windows 11...

**Comment by simonpa71** [comment]
I can confirm the issue with screen size and resolution. My  Windows 10 laptop comes with a 150% default screen resize, and some items are unusable. 
![Gittyup-150percent-20240325](https://github.com/Murmele/Gittyup/assets/4717097/8bcd482e-86ca-4488-8a18-5aa622fc6f61)

**Comment by goktugerol-dev** [comment]
seems like it's pretty much abandoned...

**Comment by FormularSumo** [comment]
Also having this issue with the window not remembering size when launched again. Similarly, it doesn't remember when I've resized the repositories list after launching again, which is a problem because the default size it too small to see the end of some longer repository names, in my case meaning I can't actually tell some of them apart (identical starts).

**Comment by AHSauge** [comment]
For Windows, #1008 should fix it if it's run outside a terminal and there's a session restore. #982 does the same thing for Linux. MacOS should already have this functionality. Now, #1013 adds a fallback if the session restore fails or it's run from the terminal.
